Global/JetBrains.gitignore

```diff
@@ -1,6 +1,9 @@
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
+# IML
```

Comment:
@shiftkey What would have in mind? Only thing I could think you might want to add is a link to something that states these shouldn't be committed.